components/MainArticle.tsx

```diff
@@ -2,7 +2,12 @@ import { Box, Flex, Image, Heading, Text } from '@chakra-ui/react';
 import { PostOrPage, Tag } from '@tryghost/content-api';
 import { urlForPost } from '../lib/posts';
 import { urlForTag } from '../lib/tags';
+
```

Comment:
future commits would be good to minimize unneeded whitespace.

Reply:
Yep. Will be mindful of that going forward.